src/Components/WebAssembly/Build/test/BuildIntegrationTests/BuildCompressionTests.cs

```diff
@@ -14,7 +14,7 @@ public class BuildCompressionTests
         public async Task Build_WithLinkerAndCompression_IsIncremental()
         {
             // Arrange
-            using var project = ProjectDirectory.Create("standalone");
+            using var project = ProjectDirectory.Create("standalone", additionalProjects: new[] { "razorclasslibrary" });
```

Comment:
This app was already being referenced by this project before this change - https://github.com/dotnet/aspnetcore/blob/blazor-wasm/src/Components/WebAssembly/Build/testassets/standalone/standalone.csproj#L17. Referencing a type from this library causes these builds to fail, so this copies it over to the build output.